sway-lib-core/src/ops.sw

```diff
@@ -472,3 +511,38 @@ impl OrdEq for u16 {
 }
 impl OrdEq for u8 {
 }
+
+/////////////////////////////////////////////////
+// Internal Helpers
+/////////////////////////////////////////////////
+
+// Extract a singe word from a b256 value using a specified offset.
```

Comment:
Triple `/` on these, since it's still docstrings that an IDE can display.

Reply:
sure.